### DISCUSSION TITLE
How do the threshholds and notifications work?

### DISCUSSION THREAD

**Discussion by shalak** [alexjustesen/speedtest-tracker#1694] [Q&A] [answered]
I configured the email notifications, test email works is received.

However - what's the purpose of this? Can I make speedtest-tracker notify me that e.g. 10 last tests were under the expected speed?

In the threshholds settings, I only see the option to notify me on each and every test that falls below expected value:

<img width="765" alt="image" src="https://github.com/user-attachments/assets/a9fb02f7-67c2-447b-8371-d4f5765f4c05">

That would be a bit noisy, as random, one-off failures to meet expected speeds is not an issue.

Does speedtest-tracker support any other threshhold types? Am I missing something?

Other the absolute threshhold, what can the notifications be used for?

**Comment by alexjustesen** [comment, marked as answer]
At this time only thresholds are supported, other types are planned after the dashboard metrics gets reworked and an API is implemented.
